src/uabytestring.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * uabytestring.cpp
+ *
+ *  Created on: Jan 26, 2017
+ *      Author: pnikiel
+ */
+
+#include <uabytestring.h>
+
+#include <open62541_compat_common.h>
+
+UaByteString::UaByteString ()
+{
+	m_impl = UA_ByteString_new();
+	if (!m_impl)
+		throw alloc_error();
+	UA_ByteString_init(m_impl);
```

Comment:
Whenever possible, I'd like to use open62541 API functions instead of looking into their data representation. Since UA_ByteString_new() and UA_ByteString_init() exist, I trust they do the proper thing. If not, we should file a bug report ;-)